tests/foreman/ui/test_hostgroup.py

```diff
@@ -57,12 +57,14 @@ def test_update_hostgroup(self, new_name):
         with Session(self.browser) as session:
             make_hostgroup(session, name=name)
             self.assertIsNotNone(self.hostgroup.search(name))
-            self.hostgroup.update(name, new_name=new_name)
-            self.assertIsNotNone(self.hostgroup.search(new_name))
+            for new_name in generate_strings_list(len1=4):
+                with self.subTest(new_name):
+                    self.hostgroup.update(name, new_name=new_name)
+                    self.assertIsNotNone(self.hostgroup.search(new_name))
+                    name = new_name  # for next iteration
 
-    @data(*generate_strings_list(len1=256))
```

Comment:
You are right, we are using it extensively using the helper invalid_values_list.  To maintain consistency, I am going to merge few tests here and use the invalid_values_list() here. do you agree?